pkg/enqueue-bundle/Tests/Unit/DependencyInjection/EnqueueExtensionTest.php

```diff
@@ -633,6 +635,31 @@ public function testShouldThrowIfPackageShouldBeInstalledToUseTransport()
         ]], $container);
     }
 
+    public function testShouldLoadProcessAutoconfigureChildDefinition()
+    {
+        $container = $this->getContainerBuilder(true);
+        $extension = new EnqueueExtension();
+
+        $extension->load([[
+            'client' => [],
+            'transport' => [],
+        ]], $container);
+
+        if (method_exists($container, 'registerForAutoconfiguration')) {
```

Comment:
The test has to be split into two. One test the case where the method is not available and whet it presents